images/base/files/usr/local/bin/entrypoint

```diff
@@ -145,6 +145,12 @@ configure_containerd() {
       systemctl enable containerd-fuse-overlayfs
     fi
   fi
+
+  if [[ -e "/etc/containerd/certs.d" ]]; then
```

Comment:
This is still technically a breaking change because the user could've been using this directory with the old config approach.

I'm not sure this is better than just announcing that we've changed the config and we've already encouraged users customizing the config to switch and updated our registry sample script to switch.